cypress.config.ts

```diff
@@ -1,9 +1,21 @@
-import { defineConfig } from "cypress";
+import { defineConfig } from 'cypress'
 
 export default defineConfig({
   e2e: {
     setupNodeEvents(on, config) {
-      // implement node event listeners here
+      on('before:browser:launch', (browser, launchOptions) => {
```

Comment:
Make sure the browser window is reasonably wide and tall enough so that tests can run without scrolling too much.